doc/introduction/interfaces.rst

```diff
@@ -3,17 +3,99 @@
 
 .. _intro_interfaces:
 
-Interfaces and training
-=======================
+Gradients and training
+======================
 
 PennyLane offers seamless integration between classical and quantum computations. Code up quantum
-circuits in PennyLane, and connect them easily to the top scientific computing and machine learning
-libraries.
+circuits in PennyLane, compute :doc:`quantum gradients <glossary/quantum_gradient>`, and
```

Comment:
```suggestion
circuits in PennyLane, compute :doc:`gradients of quantum circuits <glossary/quantum_gradient>`, and
```
just a suggestion...I realised that "quantum + word" is actually quite confusing. What is quantum about the gradient, does it contain a special kind of magic gradient? So trying to be explicit where possible.